console/src/views/DisplaySettings.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import kind from '@enact/core/kind';
+import {Panel} from '@enact/agate/Panels'
+import ColorPicker from '@enact/agate/ColorPicker';
+import {Row, Column, Cell} from '@enact/ui/Layout';
+import SliderButton from '@enact/agate/SliderButton';
+import Divider from '@enact/agate/Divider';
+import viewCss from './Settings.less';
+
+import AppContextConnect from '../App/AppContextConnect';
+import Button from '@enact/agate/Button/Button';
+
+const DisplaySettings = kind({
+	name: 'DisplaySettings',
+
+	styles: {
+		css: viewCss,
+		className: 'settingsView'
+	},
+
+	handlers: {
+		onSelect: (ev, {onSelect}) => {
+			onSelect({index: parseInt(ev.currentTarget.dataset.tabindex)});
+		}
+	},
+
+	render: ({css, onSelect, ...rest}) => (
+		<Panel {...rest}>
+			<Row className="enact-fit" align=" center">
```

Comment:
Unrelated to the data side but I'm concerned about the `Cell` usage here. It's starting to feel like `<table><tr><td>` soup for layout.